core/src/main/java/org/elasticsearch/index/query/MatchQueryBuilder.java

```diff
@@ -153,6 +160,9 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeOptionalString(fuzzyRewrite);
         out.writeOptionalWriteable(fuzziness);
         out.writeOptionalFloat(cutoffFrequency);
+        if (out.getVersion().onOrAfter(Version.V_7_0_0_alpha1)) {
```

Comment:
I will but I can't disable all bwc tests that use one of the query impacted by this change so I'll change the version after the backport.